Gemfile

```diff
@@ -65,9 +65,10 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 ## Gems manually added to for qa and qa_server engines
 # Required gems for QA and linked data access
-gem 'qa_server', '~> 7.5'
-gem 'qa', '~> 5.5'
+gem 'qa_server', git: "https://github.com/LD4P/qa_server.git", branch: 'update_rails'
```

Comment:
I'm guessing this branch is pinned in the Gemfile just for testing, but this will be updated to point to v8.0 once the update_rails branch has been merged in?